public/src/test/java/com/revenuecat/purchases/RecurrenceModeTest.kt

```diff
@@ -0,0 +1,26 @@
+package com.revenuecat.purchases
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.revenuecat.purchases.models.RecurrenceMode
+import com.revenuecat.purchases.models.toRecurrenceMode
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+ class RecurrenceModeTest {
```

Comment:
is there some way to test that we have all of the values defined in billing client?